static/js/hotkey.js

```diff
@@ -515,6 +515,12 @@ exports.process_hotkey = function (e, hotkey) {
         }
     }
 
+    // Prevents up and down navigation along the message list when lightbox is open.
+    if ((event_name === 'up_arrow' || event_name === 'down_arrow') && overlays.lightbox_open()) {
+        e.preventDefault();
```

Comment:
I think it's fine for it to scroll the overlay in view (if it's scrollable) like the up/down keys do by default; what we don't want is moving the message list below the overlay via the hotkeys code.